lib/cors.js

```diff
@@ -86,9 +86,10 @@ module.exports = function cors(config) {
         return res.status(403).send(template);
       }
 
-      const requestHeaders = (
-        req.get("access-control-request-headers") || ""
-      ).split(",");
+      const requestHeaders = (req.get("access-control-request-headers") || "")
+        .split(",")
+        .filter(h => h);
```

Comment:
I think you could make it clearer here that you're (properly) handling the edge case where the `access-control-request-headers` string is empty to begin with. Could you perhaps rewrite this as a ternary
```javascript
req.get('access-control-request-headers')
  ? req.get('access-control-request-headers').split(',')
  : [];
```
similar to how we determine `allowedHeaders`?

Reply:
No problem, done.